container_registry/container_analysis/src/sample/samples.go

```diff
@@ -0,0 +1,247 @@
+	req := &grafeaspb.CreateNoteRequest{Parent: parent, NoteId: noteID, Note: &note}
+	return client.CreateNote(ctx, req)
+}
+
+// [END create_note]
+
+// [START create_occurrence]
+
+//createsOccurrence creates and returns a new Occurrence of a previously created vulnerability Note.
+func createOccurrence(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client, imageUrl, parentNoteID, projectID string) (*grafeaspb.Occurrence, error) {
+	noteName := "projects/" + projectID + "/notes/" + parentNoteID
+	projectName := "projects/" + projectID
+
+	details := vulnerability.Details{}
+	// Details about the vulnerability instance can be added here.
+	vulDetails := grafeaspb.Occurrence_Vulnerability{Vulnerability: &details}
+	// Attach the occurrence to the associated image uri.
+	resource := grafeaspb.Resource{Uri: imageUrl}
+	occurrence := grafeaspb.Occurrence{NoteName: noteName, Resource: &resource, Details: &vulDetails}
+	req := &grafeaspb.CreateOccurrenceRequest{Parent: projectName, Occurrence: &occurrence}
+	return client.CreateOccurrence(ctx, req)
+}
+
+// [END create_occurrence]
+
+// [START update_note]
+
+// updateNote pushes an update to a Note that already exists on the server.
+func updateNote(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client, updated *grafeaspb.Note, noteID, projectID string) error {
+	noteName := "projects/" + projectID + "/notes/" + noteID
+
+	req := &grafeaspb.UpdateNoteRequest{Name: noteName, Note: updated}
+	_, err := client.UpdateNote(ctx, req)
+	return err
+}
+
+// [END update_note]
+
+// [START update_occurrence]
+
+// updateOccurrences pushes an update to an Occurrence that already exists on the server.
+func updateOccurrence(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client, updated *grafeaspb.Occurrence, occurrenceName string) error {
+	req := &grafeaspb.UpdateOccurrenceRequest{Name: occurrenceName, Occurrence: updated}
+	_, err := client.UpdateOccurrence(ctx, req)
+	return err
+}
+
+// [END update_occurrence]
+
+// [START delete_note]
+
+// deleteNote removes an existing Note from the server.
+func deleteNote(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client, noteID, projectID string) error {
+	noteName := "projects/" + projectID + "/notes/" + noteID
+
+	req := &grafeaspb.DeleteNoteRequest{Name: noteName}
+	return client.DeleteNote(ctx, req)
+}
+
+// [END delete_note]
+
+// [START delete_occurrence]
+
+// deleteOccurrence removes an existing Occurrence from the server.
+func deleteOccurrence(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client, occurrenceName string) error {
+	req := &grafeaspb.DeleteOccurrenceRequest{Name: occurrenceName}
+	return client.DeleteOccurrence(ctx, req)
+}
+
+// [END delete_occurrence]
+
+// [START get_note]
+
+// getNote retrieves a specified Note from the server.
+func getNote(ctx context.Context, client *containeranalysis.GrafeasV1Beta1Client, noteID, projectID string) (*grafeaspb.Note, error) {
```

Comment:
Do you want to print out some details of the note here instead of just returning it? Like you do for list occurrences? Otherwise this just becomes a wrapper around the API call. Same with other get methods.